burst/burst.py

```diff
@@ -198,7 +198,7 @@ def got_results(provider, results):
     # 2 "Size"
     # 3 "Balanced"
 
-    if not sort_by or sort_by == 3:
```

Comment:
This is a check of empty "sort_by", why removing this?

Reply:
@elgatito Reason is because AFAICT `get_setting('sort_by', int)` never can return an empty value. It will  throw a exception instead.